testsuite/features/secondary/proxy_retail_pxeboot_and_mass_import.feature

```diff
@@ -189,7 +189,7 @@ Feature: PXE boot a Retail terminal
     And I follow the left menu "Systems > Overview"
     And I wait until I see the name of "pxeboot_minion", refreshing the page
     And I follow this "pxeboot_minion" link
-    And I wait until event "Apply states [util.syncstates, saltboot] scheduled by (none)" is completed
+    And I wait at most 350 seconds until event "Apply states [util.syncstates, saltboot] scheduled by (none)" is completed
```

Comment:
is this the only place where this is necessary? I thought there were several steps/ scenarios affected :thinking:

Reply:
Correct. I was not sure if we decided to put it into more places.

Reply:
I increased the timeout for the bootstrapping the PXE boot minion, too. I think the rest of the steps fail due to those failures.